api/migrations/0003_fake_data.py

```diff
@@ -50,7 +66,27 @@ def _get_random_amount_of_objects(
             length=self.faker.pyint(min_value=min_length, max_value=max_length),
         )
 
-    def _make_personal_info_recipe(self):
+    def _make_group_common_recipe(
+        self, model_name: str, teachers_min_value: int, teachers_max_value: int
+    ) -> Recipe:
+        return Recipe(
+            model_name,
+            coordinators=lambda: related(
+                *[self.coordinator_recipe] * self.faker.pyint(min_value=1, max_value=5)
+            ),
+            students=lambda: related(
+                *[self.student_recipe] * self.faker.pyint(min_value=5, max_value=25)
+            ),
+            teachers=lambda: related(
+                *[self.teacher_recipe]
+                * self.faker.pyint(
+                    min_value=teachers_min_value, max_value=teachers_max_value
+                )
+            ),
```

Comment:
a group usually has one teacher, in rare cases two teachers. Same for coordinators.
number of students is 2 to 10 people, as per the owners' poll. I think it makes sense to follow these guidelines for the dashboard to look real right from the start.

Reply:
Makes sense. Fixed